vscode/src/extension.ts

```diff
@@ -1384,8 +1392,8 @@ function doActivateWithJDK(specifiedJDK: string | null, context: ExtensionContex
                     }
                 });
             } else {
-                const yes = "Install GPLv2+CPEx code";
-                window.showErrorMessage("Additional Java Support is needed", yes).then(reply => {
+                const yes = l10n.getLocaleValue("jdk.extension.javaSupport.label.installGpl");
+                window.showErrorMessage(l10n.getLocaleValue("jdk.extension.javaSupport.message.needAdditionalSupport"), yes).then(reply => {
                     if (yes === reply) {
                         vscode.window.setStatusBarMessage(`Preparing ${SERVER_NAME} for additional installation`, 2000);
```

Comment:
Please add the status bar message for localisation.

And, also the error on line 1484:
`'Oracle Java SE Debug Server Adapter not yet initialized. Please wait for a while and try again.'`